src/main/java/seedu/tp/flashcard/Flashcard.java

```diff
@@ -133,6 +134,15 @@ public String getSummary() {
         return details;
     }
 
+    /**
+     * To be implemented by child classes.
+     *
+     * @return a shortened description of the flashcard
+     */
+    public String getShortDescription() {
```

Comment:
Consider making this an abstract method to force all child classes to implement it since getting an empty short description doesn't really make sense for any flashcard?